app/assets/javascripts/edit_utils.js

```diff
@@ -16,9 +16,7 @@ Vite wraps it as an ESM, and there doesn't seem to be a way to specify
 an export that we can call as-needed.
 */
 
-console.log('edit_utils.js loaded...');
 $(() => {
-  console.log('edit_utils.js hooks loading...');
```

Comment:
Perhaps also https://developer.mozilla.org/en-US/docs/Web/API/console/debug might permit this to remain?